src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -51,6 +62,14 @@ public class PersonCard extends UiPart<Region> {
     public PersonCard(Person person, int displayedIndex) {
         super(FXML);
         this.person = person;
+
+
+        Image studentImage =
+                new Image(Objects.requireNonNull(this.getClass().getResourceAsStream("/images/student.png")));
```

Comment:
Can we try to avoid hardcoding this image input path?

Reply:
Sure, I hardcoded it as we have not developed the Student class yet. So we shall leave it until the Student class is created